src/main/kotlin/no/nav/syfo/melding/kafka/domain/DialogmeldingType.kt

```diff
@@ -9,6 +9,7 @@ enum class DialogmeldingType {
 }
 
 fun DialogmeldingType.getMeldingType(): MeldingType {
+    // TODO: Her må vi også håndtere legeerklæring
     return when (this) {
         DialogmeldingType.DIALOG_SVAR -> MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER
         else -> throw IllegalArgumentException("Cannot get MeldingType for DialogmeldingType $this")
```

Comment:
Her må vi finne en måte å koble svaret vi får inn til riktig type. Både tilleggsopplysninger og legeerklæring får svar av typen `DIALOG_SVAR`, så kanskje vi må se på tidligere meldinger i conversationRef for å knytte det til riktig type?